social/author/templates/author/index.html

```diff
@@ -25,6 +25,9 @@
             <div class="nav_panel">
                 Navigate
                 <div class="nav_item">
+                    <a href="/a/profile/">My Profile</a>
```

Comment:
Probably better to use Jinja url fields here so they aren't broken if the /a/ path changes.